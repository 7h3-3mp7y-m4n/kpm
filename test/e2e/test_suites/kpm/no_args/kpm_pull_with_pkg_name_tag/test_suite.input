kpm pull k8s:1.27.2