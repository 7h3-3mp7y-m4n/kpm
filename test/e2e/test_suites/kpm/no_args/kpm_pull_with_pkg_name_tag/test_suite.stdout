kpm: start to pull 'k8s:1.27'.
kpm: pulling 'kcl-lang/k8s:1.27' from 'ghcr.io/kcl-lang/k8s'.
kpm: pulled 'k8s:1.27' in '<workspace>/ghcr.io/kcl-lang/k8s/1.27' successfully.
